core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
@rabbah, i tested, when use api and directly acess to controller, you said above will happen.
```
$ curl -X GET --header 'Accept: application/json' \
> --header 'x-ssl-subject: CN=guest,O=OpenWhisk,L=Yorktown,ST=NY,C=US' \
> --header 'namespace: guest' \
>  'http://<my_local_ip>:10001/api/v1/namespaces/guest/actions'
[]
```
So do you have any idea how to add the guard?